hack/remove-calico-policy/README.md

```diff
@@ -0,0 +1,13 @@
+# Disabling and removing Calico policy
+
+1. Stop and remove the calico-node Pod on each Node to prevent network policy from being (re)programmed.
```

Comment:
We should note there that there are a number of ways to do this - the DaemonSet approach is not completely ubiquitous. 

Some others:
- systemd
- DaemonSet via add-on manager